### PR TITLE
All route to single action

### PR DESCRIPTION
**Pull request by tejaswinichile**
Issue #395

### CODE REVIEW

File: app/javascript/Admin/Main.elm [hunk 1]
```diff
@@ -1031,9 +1003,6 @@ view model =
         NotFound ->
             Errors.notFound
 
-        Blank ->
```
Comment (breezyboa) [inline app/javascript/Admin/Main.elm]: We need a Blank page to display as the app loads up this cannot be removed

Comment (tejaswinichile) [reply app/javascript/Admin/Main.elm]: Okay 👍

**Comment by tejaswinichile** [comment]
@breezyboa _a

**Comment by breezyboa** [comment]
@abdul-shajin _a can you review rails code

**Pull request merged**